install/tests/Makefile

```
@@ -92,6 +124,10 @@ check-gitpod-installation: delete-cm-setup check-kots-app check-env-sub-domain
 run-tests:
 	./tests.sh ${KUBECONFIG}
 
+kots-uprgade:
```

```suggestion
kots-upgrade:
```

(Tiny nitpick, I think this may have already been addressed in a previous PR)